src/chainparams.cpp

```diff
@@ -388,11 +393,11 @@ class CRegTestParams : public CChainParams {
         consensus.nRuleChangeActivationThreshold = 108; // 75% for testchains
         consensus.nMinerConfirmationWindow = 144; // Faster than normal for regtest (144 instead of 2016)
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].bit = 28;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nStartTime = 0;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT;
+        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].startheight = 0;
```

Comment:
`startheight = 144` would be more accurate, it can't actually start at height 0.

(This caused a weird bug in `validation_blocks_tests` when I tried implementing bitcoin/bips#1023 and it was able to go to STARTED in the first block; didn't track down exactly what the issue was though)